.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
- - [ ] closes #xxxx
- - [ ] tests added / passed
- - [ ] passes ``git diff upstream/master -u -- "*.py" | flake8 --diff``
- - [ ] whatsnew entry
+- [ ] closes #xxxx
+- [ ] tests added / passed
+- [ ] passes `git diff upstream/master -- "*.py" | flake8 --diff`
```

Comment:
I would refrain from changing the `git diff` checks.  Unless you can confirm that these work across all platforms (we've run into this problem before), that should not be touched.